apps/prairielearn/src/lib/db-types.ts

```diff
@@ -239,6 +239,7 @@ export type Question = z.infer<typeof QuestionSchema>;
 
 export const WorkspaceSchema = z.object({
   created_at: DateFromISOString,
+  disk_usage_bytes: z.coerce.number().nullable(), // This is BIGINT, but always fits a number
```

Comment:
Right now `getDirectoryDiskUsage` returns a number. If you want it to be treated as a bigint, you should probably change that calculation to also use bigint, and change the call to `lstat` to add `{ bigint: true }`. This is probably not necessary, unless you expect a workspace to use values in the order of petabytes, and even in that case I don't think having an accurate number of bytes matters to you. The number type is likely good enough here.

Reply:
Yeah, if we have petabytes of data in a workspace, we have a far bigger problem. `bigint` was chosen for the DB column because just `integer` isn't large enough (we already have some workspaces larger than that). Most reasonable Postgres `bigint` values can fit in a standard JavaScript number, so we should be safe.